src/borg/fuse.py

```diff
@@ -158,36 +307,31 @@ def process_archive(self, archive_name, prefix=[]):
         """
         self.file_versions = {}  # for versions mode: original path -> version
         t0 = time.perf_counter()
-        unpacker = msgpack.Unpacker()
         archive = Archive(self.repository_uncached, self.key, self.manifest, archive_name,
                           consider_part_files=self.args.consider_part_files)
-        for key, chunk in zip(archive.metadata.items, self.repository.get_many(archive.metadata.items)):
-            data = self.key.decrypt(key, chunk)
-            unpacker.feed(data)
-            for item in unpacker:
-                item = Item(internal_dict=item)
-                path = os.fsencode(os.path.normpath(item.path))
```

Comment:
I removed the normpaths since it's a rather expensive operation that is already always done by `borg create` anyway.